### PR TITLE
Update Storage libraries for accurate released packages

### PR DESCRIPTION
**Pull request by amnguye**
To resolve https://github.com/Azure/azure-sdk-for-net/issues/22704

### CODE REVIEW

**Comment by amnguye** [comment]
@tamram @CamSoper @jsquire @scottaddie Could I get someone to review this? I don't believe I have permissions to add reviewers.

File: api/overview/azure/storage.md [hunk 1]
```diff
@@ -25,7 +25,6 @@ Use the following version 12.x libraries to work with blobs, files, and queues:
 | Library | Reference | Package | Source |
 |----------------------------------------|-------------------------------------------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
 |    Azure.Storage.Blobs.Batch    |         |    [NuGet](https://www.nuget.org/packages/Azure.Storage.Blobs.Batch/)    |    [GitHub](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs.Batch)    |
```
Comment (jsquire) [inline api/overview/azure/storage.md:27]: Should we replace `master` with `main` to avoid the redirect?

**Review by jsquire — APPROVED** [review]

File: api/overview/azure/storage.md (hunk 1 above)
Comment (amnguye) [reply api/overview/azure/storage.md:27]: Good catch

Comment (amnguye) [reply api/overview/azure/storage.md]: done changed it.

**Comment by amnguye** [comment]
Can someone also merge this if this PR is good enough? I don't have merge permissions on this repo. Thanks!

**Pull request merged**